libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -71,10 +72,11 @@
 #define CONFIG_NAV_SETTINGS  (1<<10)
 #define CONFIG_GNSS          (1<<11)
 #define CONFIG_SBAS          (1<<12)
+#define CONFIG_RATE_PVT      (1<<13)
 
 #define CONFIG_ALL (CONFIG_RATE_NAV | CONFIG_RATE_POSLLH | CONFIG_RATE_STATUS | CONFIG_RATE_SOL | CONFIG_RATE_VELNED \
                     | CONFIG_RATE_DOP | CONFIG_RATE_MON_HW | CONFIG_RATE_MON_HW2 | CONFIG_RATE_RAW | CONFIG_VERSION \
-                    | CONFIG_NAV_SETTINGS | CONFIG_GNSS | CONFIG_SBAS)
+                    | CONFIG_NAV_SETTINGS | CONFIG_GNSS | CONFIG_SBAS | CONFIG_RATE_PVT)
```

Comment:
:/ I've fixed that twice now and apparently lost it via bad usage of git stash. PVT shouldn't be in that mask. (Alternatively it can be, but you have to check if the POSLLH/VELNED/STATUS aren't. It's easier to just not add it to that mask.